### PR TITLE
Set an empty collection on null target fields when copying translation data

### LINKED ISSUE #67: Translation fails if relation is handled by the translation entity
Assuming you have an entity with a relation handled by it's translation, it will fail during save if the value for the field of that relation is null. Of course initially you need to pass an array or an `ArrayCollection` instance to the file in the constructor, which solves the problem during creation. The issue arises in a situation when:

1. There are no objects in the said relation.
2. The translated object is fetched via the entity manager (so the constructor is omitted).
3. You edit the object through a form where the field for that relation does not exist.

This will result in a null value being copied to the translation class's field. To get around this, you need to add a lifecycle callback (like `postLoad`) to the translated entity, that will properly set the value.

This is not a huge issue and it does not require a hack to get around, but perhaps we could check for a null value and if the field to which it is copied is a relation, set an empty array instead?

### PR DESCRIPTION
**Pull request by szymach**
Resolves #67 

TODO:
- [x] Remove docs entries regarding using `postLoad` to fix this issue

### CODE REVIEW

**Comment by szymach** [comment]
@fsi-open thoughts?

File: lib/FSi/DoctrineExtensions/Translatable/TranslationHelper.php [hunk 1]
```diff
@@ -76,24 +81,26 @@ public function copyPropertiesToTranslation(
...
-    ) {
+    public function removeEmptyTranslation(ClassTranslationContext $context, $object)
+    {
         if ($this->hasTranslatedProperties($context, $object)) {
             return;
         }
 
         $objectLocale = $this->getObjectLocale($context, $object);
-        if (!isset($objectLocale)) {
+        if (is_null($objectLocale) || $objectLocale === '') {
```
Comment (JarekW) [inline lib/FSi/DoctrineExtensions/Translatable/TranslationHelper.php]: test

**Comment by szymach** [comment]
@JarekW @rn0

**Comment by szymach** [comment]
@rn0

**Comment by szymach** [comment]
@rn0

**Pull request merged**